kyo-cats/shared/src/main/scala/kyo/Cats.scala

```diff
@@ -0,0 +1,46 @@
+package kyo
+
+import Cats.GetCatsIO
+import cats.effect.IO as CatsIO
+import kyo.kernel.*
+import scala.util.control.NonFatal
+
+opaque type Cats <: (Async & Abort[Throwable]) = GetCatsIO & Async & Abort[Throwable]
+
+object Cats:
+
+    /** Lifts a cats.effect.IO into a Kyo effect.
+      *
+      * @param io
+      *   The cats.effect.IO to lift
+      * @return
+      *   A Kyo effect that, when run, will execute the cats.effect.IO
+      */
+    def get[A](io: CatsIO[A])(using Frame): A < (Abort[Throwable] & Cats) =
```

Comment:
Should `io` be by name?
Does the return type need the `Abort`? I guess that is more clear for users.

Reply:
It doesn't need the `Abort` since it's in the opaque type. We don't strictly need a by-name param but that's the pattern in `ZIOs` and it'd make error handling better in case the `io` construction fails. I'll update it.